flow/scenarios/base_scenario.py

```diff
@@ -58,6 +58,243 @@ class Scenario(Serializable):
+        * **to**: the name of the node the edges ends at
+        * **length**: length of the edge
+
+        In addition, either the following properties need to be specifically
+        defined or a **type** variable property must be defined with equivalent
+        attributes in `self.types`:
+
+        * **numLanes**: the number of lanes on the edge
+        * **speed**: the speed limit for vehicles on the edge
+
+        Moreover, the following attributes may optionally be available:
+
+        * **shape**: the positions of intermediary nodes used to define the
+          shape of an edge. If no shape is specified, then the edge will appear
+          as a straight line.
+
+        Note that, if the scenario is meant to generate the network from an
+        OpenStreetMap or template file, this variable is set to None
+    types : list of dict or None
+        A variable used to ease the definition of the properties of various
+        edges. Each element in the list consists of a dict consisting of the
+        following property:
+
+        * **id**: name of the edge type. Edges in the `self.edges` attribute
+          with a similar value under the "type" key will adopt the properties
+          of other components of this list, such as "speed" and "numLanes".
+
+        If the type variable is None, then no types are available within the
+        scenario.
+
+        Note that, if the scenario is meant to generate the network from an
+        OpenStreetMap or template file, this variable is set to None
+    connections : list of dict or None
+        TODO
+
+        If the connections attribute is set to None, then the connections
+        within the network will be specified by the simulator.
+
+        Note that, if the scenario is meant to generate the network from an
+        OpenStreetMap or template file, this variable is set to None
+    routes : dict
+        A variable whose keys are the starting edge of a specific route and the
+        element is the list of edges a vehicle is meant to traverse starting
+        from that edge. These are only applied at the start of a simulation;
+        vehicles are allowed to reroute within the environment immediately
+        afterwards.
+    edge_starts : list of (str, float)
+        a list of tuples in which the first element of the tuple is the name of
+        the edge/intersection/internal_link, and the second value is the
+        distance of the link from some global reference, i.e. [(link_0, pos_0),
+        (link_1, pos_1), ...]
+    internal_edge_starts : list of (str, float)
+        A variable similar to `self.edge_starts` but for junctions within the
+        network. If no junctions are available, this variable will return the
+        default variable: `[(':', -1)]` needed by sumo simulations.
+    intersection_edge_starts : list of (str, float)
+        A variable similar to `self.edge_starts` but for intersections within
```

Comment:
I think it would be more clear to just put 'edge_starts', so that it's clear you're referring to the same 'edge_starts' in line 150.

Reply:
done